frontend/src/pages/PersonalPage.jsx

```diff
@@ -226,18 +234,31 @@ function PersonalPage(props) {
                             <SectionTitle title="Biography" />
                             <div className="agenda big">{userDescription}</div>
                         </div>
-
-                        <div className="d-flex column right-side-middle">
+                        <div className="d-flex flex-column align-items-start mynote-results-container">
                             <SectionTitle title="My Notes" />
-                            {/* <PDFViewer thumbnail pdf={pdf} /> */}
-                            <div className="d-flex flex-row right-side-down">
-                                <Button
-                                    title="UPLOAD"
-                                    type="primary"
-                                    clicked={() => setButtonUpload(true)}
-                                />
+                            <div className="d-flex flex-row flex-wrap">
+                                {pdf.map((eachDPF) => (
+                                    <Link
+                                        to={"/note/" + eachDPF.note_id}
+                                        style={{
+                                            color: "inherit",
+                                            textDecoration: "inherit",
+                                        }}
+                                    >
+                                        <PersonalPagePDFViewer
```

Comment:
I think we can add an onclick function to the pdf viewer instead of wrapping it with a link, but that's personal preference